dvc/main.py

```diff
@@ -67,7 +67,9 @@ def main(argv=None):
             )
         elif isinstance(exc, OSError) and exc.errno == errno.EMFILE:
             logger.exception(
-                "too many open files, please increase your `ulimit`"
+                "too many open files, please visit "
+                "https://dvc.org/doc/user-guide/troubleshooting to see how "
```

Comment:
Please use `format_link` helper from utils.

Reply:
The link doesn't work yet, but when it will, it will probably have a separate section for that, right? If so, you need to use section link.